syntax = "proto3";

package avninv.serde.test;

message _TestMessage {
    message _NestedTestMessage {
        bool bool_field_1 = 1;
        int64 int64_field_2 = 2;
    }

    string string_field_1 = 1;
    uint64 uint64_field_2 = 2;
    _NestedTestMessage nested_field_3 = 3;

    repeated _NestedTestMessage repeated_nested_field_4 = 4;
    repeated string repeated_string_field_5 = 5;
}